src/MunicipalityRegistry.Projections.Extract/Migrations/20201023084133_AddErrorMessage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace MunicipalityRegistry.Projections.Extract.Migrations
+{
+    public partial class AddErrorMessage : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "ErrorMessage",
+                schema: "MunicipalityRegistryExtract",
+                table: "ProjectionStates",
+                nullable: true);
+
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "DbaseRecord",
+                schema: "MunicipalityRegistryExtract",
+                table: "Municipality",
+                nullable: false,
```

Comment:
@ArneD what happened here? Any reason why this column should not be nullable anymore, is this intentional or an accident?